RLTsquare/CustomRoute/etc/email_templates.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Email:etc/email_templates.xsd">
+    <template id="custom_template" label="Welcome Template" file="welcome.html" type="html" module="RLTsquare_CustomRoute" area="frontend"/>
```

Comment:
Naming convention - we should not use "custom_template" ... . we should adjust the naming properly to match our module, also this naming convention suggest you fif not use system.xml and config.xml to make template configurable from admin. See https://drive.google.com/file/d/1ed3KeOMXJoXriWg7CJeEh4rEGzFkU1vk/view?usp=share_link